### PR TITLE
194 file structure

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#194 

---

#### Any info the reviewer needs to know to test 
- npm i

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: server/services/validation/fields.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * basic validation schema for common fields
+ */
+import { string } from 'yup';
+
+export const name = string().min(3).max(20);
```
Comment (fadeomar) [inline server/services/validation/fields.js:6]: there is no name in Figma, instead, there is password

**Comment by thejoefriel** [comment]
you're right. in figma there is email & password for log in

And then there's first name last name for sign up.

File: server/services/validation/fields.js (hunk 1 above)
Comment (ashatat) [inline server/services/validation/fields.js:6]:
```suggestion
export const firstName = string().min(3).max(20);
```

Comment (RamyAlshurafa) [reply server/services/validation/fields.js]: done

**Pull request merged**